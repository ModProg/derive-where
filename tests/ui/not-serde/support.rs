use std::marker::PhantomData;

use derive_where::derive_where;

#[derive_where(Deserialize)]
struct UnsupportedDeserialize<T>(PhantomData<T>);

#[derive_where(Serialize)]
struct UnsupportedSerialize<T>(PhantomData<T>);

fn main() {}
