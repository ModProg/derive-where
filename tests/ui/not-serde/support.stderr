error: requires crate feature `serde`
 --> tests/ui/not-serde/support.rs:5:16
  |
5 | #[derive_where(Deserialize)]
  |                ^^^^^^^^^^^

error: requires crate feature `serde`
 --> tests/ui/not-serde/support.rs:8:16
  |
8 | #[derive_where(Serialize)]
  |                ^^^^^^^^^
